homie-influx/README.md

```diff
@@ -54,6 +54,17 @@ You may find it helpful to watch the logs to see whether it is managing to conne
 $ sudo journalctl -u homie-influx.service --output=cat --follow
 ```
 
+
```

Comment:
Nit: here and below, one blank line should be enough. (We should probably run prettier in CI.)

Reply:
Done myself.